docs/source/resilience.rst

```diff
@@ -48,11 +48,12 @@ This has some fail cases.
     causes a segmentation fault, then that bad function will repeatedly be
     called on other workers.  This function will be marked as "bad" after it
     kills a fixed number of workers (defaults to three).
-3.  Data scattered out to the workers is not kept in the scheduler (it is
-    often quite large) and so the loss of this data is irreparable.  You may
-    wish to call ``Client.replicate`` on the data with a suitable replication
-    factor to ensure that it remains long-lived or else back the data off of
-    some resilient store, like a file system.
+3.  Data sent out directly to the workers via a call to ``scatter()`` (instead
+    of being created via a call to ``persist()`` from a Dask task graph) is not
```

Comment:
This parenthetical is slightly troublesome because the `persist` function is only one of a few methods that will trigger computation (client.compute, submit, map, ...)